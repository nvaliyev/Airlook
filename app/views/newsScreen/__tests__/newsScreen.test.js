import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NewsScreen from '..';
describe(' NewsScreen', () => {
    test(' NewsScreen snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< NewsScreen/>);
        expect(snapshot).toMatchSnapshot();
    });
});


