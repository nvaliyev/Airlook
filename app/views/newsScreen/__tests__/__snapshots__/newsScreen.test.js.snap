// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` NewsScreen  NewsScreen snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text>
      NewsScreen  
  </Text>
  <ClickButton
    backgroundColor="#5d6fc4"
    onPress={[Function]}
    text="Details"
  />
</View>
`;
